.github/workflows/tests.yml

```diff
@@ -31,7 +37,7 @@
 
         - name: Install dependencies
           run: |
-            composer require "illuminate/support:${{ matrix.laravel }}" --no-interaction --no-update
+            composer require "laravel/framework:${{ matrix.laravel }}" --no-interaction --no-update
```

Comment:
replacing the illuminate packages with laravel/framework, as we need additional classes and it's easier to have one dependency